packages/css/src/base.css

```diff
@@ -49,16 +49,16 @@ body,
    * We use both box-shadow and outline to ensure that the focus style is visible,
    * in case box-shadow is overridden.
    */
-  box-shadow: var(--ds--focus, var(--dsc-focus-boxShadow));
-  outline: var(--ds--focus, var(--dsc-focus-outline));
-  outline-offset: var(--ds--focus, var(--dsc-focus-border-width));
+  box-shadow: var(--_ds--focus, var(--dsc-focus-boxShadow));
+  outline: var(--_ds--focus, var(--dsc-focus-outline));
+  outline-offset: var(--_ds--focus, var(--dsc-focus-border-width));
```

Comment:
These are variables we want to allow the user to change. Are we sure we should mark these as internal?

Reply:
Or is it the variables on line 17-19 we want consumers to change?

Reply:
--_ds-focus is for us only, but `--dsc-focus-boxShadow`, `--dsc-focus-outline`, `--dsc-focus-border-width` is for the user to change ☺️